cmd/validate_image_test.go

```diff
@@ -22,23 +22,29 @@ import (
 	"errors"
 	"testing"
 
-	"github.com/hacbs-contract/ec-cli/internal/policy"
 	"github.com/open-policy-agent/conftest/output"
```

Comment:
Would make it explicit

```suggestion
	conftestoutput "github.com/open-policy-agent/conftest/output"
```